### ISSUE TITLE
Add a "Konto löschen" button to the user settings page

### ISSUE THREAD

**Issue by PiaSophieBe** [remsfal/remsfal-frontend#70]
Because delete account is probably not used frequently, it should be integrated into the account settings and not remain in the navigation. This would be in line with the standard compared to other websites.

**Comment by astanik** [comment]
I agree. In [AccountSettingsView](https://github.com/remsfal/remsfal-frontend/blob/main/src/views/AccountSettingsView.vue) there should be a button that opens a model where the user has to confirm the deletion. We should also remove with this issue the LeoModal.vue and use the Sakai Confirmation Delete overlay...
https://sakai.primevue.org/uikit/overlay
<img width="1287" alt="Bildschirmfoto 2024-06-05 um 08 59 04" src="https://github.com/remsfal/remsfal-frontend/assets/7569599/54bd6fca-bfe9-4137-a908-4beab713c7f8">

**Comment by ckc-efehan** [comment]
Ich würde das gerne machen.
